soc/nxp/imx/imx9/imx95/m7/soc.c

```diff
@@ -69,7 +74,18 @@ static int soc_init(void)
 		return ret;
 	}
 #endif
-	return 0;
+
+#if defined(CONFIG_NXP_SCMI_CPU_DOMAIN_HELPERS)
+	cpu_cfg.cpu_id = CPU_IDX_M7P;
+	cpu_cfg.sleep_mode = CPU_SLEEP_MODE_RUN;
+
+	ret = scmi_cpu_sleep_mode_set(&cpu_cfg);
```

Comment:
can you please provide a more detailed commit description on this? it is very important to understand why we need this change as it will affect multiple setups.